src/components/Users/UserAddEditForm.tsx

```
@@ -0,0 +1,1173 @@
+          }
+          break;
+        case "local_body":
+          if (showLocalbody && !Number(formData[field])) {
+            errors[field] = t("please_select_localbody");
+          }
+          break;
+        case "weekly_working_hours":
+          if (formData[field] !== null && formData[field] !== undefined) {
+            const hours = Number(formData[field]);
+            if (
+              isNaN(hours) ||
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)